src/persist/src/nemesis/mod.rs

```diff
@@ -286,57 +286,84 @@ pub struct Runner<R: Runtime> {
 
 impl<R: Runtime> Runner<R> {
     const MAX_OUTSTANDING: usize = 10;
+    // TODO: Increase this to 3 once Validator can handle that.
+    const NUM_WORKER_THREADS: usize = 1;
```

Comment:
nit: I wish there was some way to call this `NUM_INGEST_WORKER_THREADS` and then also specify `NUM_DATAFLOW_WORKER_THREADS` (or `NUM_COMPUTE_WORKER_THREADS`) at this level too? I can't tell if wanting this reflects some fundamental misunderstanding on my part like "the various implementers of `Runtime` like `Direct` get to decide what they are, so the concept of 'dataflow workers' doesn't make sense at this stage?

Reply:
no you're totally right. i think there's a good possibility that the ingestd workers are at the wrong level of abstraction and should be pushed down inside Direct (where the dataflowd workers already are). however, if you don't mind, in the interest of making progress on this, I'd like to leave this wart

Reply:
of course that's totally fine!